services/alertmanager/alertmanager.go

```diff
@@ -29,16 +29,19 @@ import (
 	"github.com/percona/pmm/api/alertmanager/amclient/alert"
 	"github.com/percona/pmm/api/alertmanager/amclient/general"
 	"github.com/percona/pmm/api/alertmanager/ammodels"
+	"github.com/percona/promconfig/alertmanager"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"gopkg.in/reform.v1"
+	"gopkg.in/yaml.v2"
```

Comment:
@artemgavrilov See? I wonder if we could something about that later

Reply:
I expect to have some free time in this sprint, so I could try to migrate everything to `v3`.

Reply:
Looks like it's not a big deal